src/@next/IndexTable/components/LoadingState/LoadingStateStyle.ts

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import { Device } from '../../../../Utils/StyleConfig';
+
+export const LoadingStateContainer = styled.td`
+  height: 480px;
+`;
+
+export const SpinnerContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
+  gap: 16px;
+
+  p {
+    margin: 0;
+  }
+
+  @media ${Device.desktopS} {
```

Comment:
I think we should use Breakpoints from `@next`

Reply:
The table is a quite wide component, Device with width of 1024 can't show the table fully and thus I need the 1024 breakpoints. Can I add a new one or do I need to stick with 768?

Reply:
@ganiirsyadi sure, but please discuss with Maeve, I think there will be duration we can't see it fully, that expected, which a scrollbar will appear

Reply:
Yes a scrollbar will appear, but I need to make sure the spinner is at the center of the window (not at the center of the table, because the table width > screen width. 

Update:
It turned out, it is safe to remove the @media query and apply those styles directly. I will ask Maeve to re-test it to make sure it doesn't break on larger device.

Reply:
> Yes a scrollbar will appear, but I need to make sure the spinner is at the center of the window (not at the center of the table, because the table width > screen width

Hmm i think the loader should be in the center of the table, since it's loading state for the table 🤔

Reply:
The best that I can do for now is to use breakpoint large to avoid unexpected behavior on Desktop screen when forcing the loading to be at the center of the screen.